assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -6,18 +6,32 @@ import isShallowEqual from '@wordpress/is-shallow-equal';
 import { savePreviewMeta } from '../../saveMetaToPreview';
 import { PostParentLink } from './PostParentLink';
 import { LegacyThemeSettings } from './LegacyThemeSettings';
+import { NewThemeSettings } from './NewThemeSettings';
+
+const isLegacy= theme => [
+  'default',
+  'antarctic',
+  'arctic',
+  'climate',
+  'oceans',
+  'oil',
+  'plastic',
+  'forest',
+].includes(theme) || !theme;
 
 const loadTheme = async (value) => {
   if ( value === '' || !value ) {
     value = 'default';
   }
+  const withoutNew = value.replace(/-new$/, '');
```

Comment:
Similar logic for the back end was already merged into master theme https://github.com/greenpeace/planet4-master-theme/blob/5c49b8a9bdc5d58bff429ed4c1446db4eacac964/src/PostCampaign.php#L440-L452